lib/stripe/util.rb

```diff
@@ -2,6 +2,8 @@
 
 module Stripe
   module Util
+    @invalid_params = Set.new [:idempotency_key, :api_key, :stripe_account]
```

Comment:
Oh, and can we also alphabetize the keys in the list and also move it to the `private` section of the class? You can use `private_const :CONST` to protect it.